app/angular/src/client/preview/angular/decorators.ts

```diff
@@ -0,0 +1,21 @@
+import { NgModuleMetadata } from './types';
+
+export const moduleMetadata = (metadata: Partial<NgModuleMetadata>) => (storyFn: () => any) => {
+  const story = storyFn();
+  const storyMetadata = story.moduleMetadata || {};
+  metadata = metadata || {};
```

Comment:
Can the default be introduced in the method signature?

Reply:
That's how I did it originally, but default params affect undefined or no value, but not null. So I'd need to add a check for null and assignment to `{}` anyway.